packages/loaders/src/styled/StyledProgress.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+type SIZE = 'small' | 'medium' | 'large';
+
+const HEIGHTS: Record<string, number> = {
+  small: 2,
+  medium: 6,
+  large: 12
```

Comment:
How do these values relate to the DEFAULT_THEME? Are they based on `space` units? If so (similar to grid), these should be pulled (or calculated on `base`) accordingly.

Reply:
We shouldn't be using DEFAULT_THEME to calculate these since that wouldn't make the values updated if `base` is themed, right? I've moved this to a utility method which uses the current theme to calculate the values from `theme.space.base`